profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module

```diff
@@ -82,12 +82,13 @@ function nexteuropa_varnish_cache_purge_rule_label($purge_rule) {
  * Implements hook_workbench_moderation_transition().
  */
 function nexteuropa_varnish_workbench_moderation_transition($node, $previous_state, $new_state) {
-  $purge
-    = ($new_state == workbench_moderation_state_published()) ||
-      (property_exists($node->workbench_moderation['my_revision'], 'unpublishing')
-        && $node->workbench_moderation['my_revision']->unpublishing);
+  $publishing = (($new_state == workbench_moderation_state_published()) &&
```

Comment:
@voidtek, is it normal that when I have a published revision of a node and I only save a Draft revision of it, a "clearing" request is sent to Varnish?
I expected that in this case, no request is sent.